.npmrc

```diff
@@ -1,5 +1,4 @@
 save-exact = true
 engine-strict = true
 unsafe-perm = true
-legacy-peer-deps = true
-frozen-lockfile = false
+strict-peer-dependencies=true
```

Comment:
Unmet peer dependency warnings thrown by pnpm were ignored completely. Now with `strict-peer-dependencies` enabled, it will be handled stricter: https://pnpm.io/npmrc#strict-peer-dependencies